pom.xml

```diff
@@ -143,6 +143,7 @@
             <plugin>
                 <groupId>org.jacoco</groupId>
                 <artifactId>jacoco-maven-plugin</artifactId>
+                <version>${jacoco-maven-plugin.version}</version>
```

Comment:
Doesn't version need defining? Like for maven-surefire-plugin.version?

Reply:
it seems to be populated (0.8.4) when hovering over in intellij, assumed it is imported from parent pom, i can check

Reply:
Will assume it's good and approve. We'll need to merge anyway to see if it fixes things.